### ISSUE TITLE
Selenium Grid - Unable to connect with selenium Grid Hub url

### ISSUE THREAD

**Issue by elumalaik** [microsoft/playwright#9719]
**Context:**
- Playwright Version: 1.16.0
- Operating System: Windows
- Node.js version: v14.17.3
- Browser: Chrome

**Code:**
 
```js
test('Playwright Demo', async ({ page }) => {
      await page.goto('https://playwright.dev/');
      await page.locator('text=Get started').click();
      await expect(page).toHaveTitle(/Getting started/);
    }); 
```

**Run:** (Via Bash)

SELENIUM_REMOTE_URL=http://ipaddress:4444/wd/hub npx playwright test

```
**Error:**
browserType.launch: Error connecting to Selenium at http://ipaddress:4444/wd/hub/:

at Object._baseTest.extend.browser.scope [as fn] (...playwrightdemo\node_modules\@playwright\test\lib\index.js:221:40)     
        at Fixture.setup (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:74:73)
        at FixtureRunner.setupFixtureForRegistration (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:315:5)       
        at Fixture.setup (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:58:19)
        at FixtureRunner.setupFixtureForRegistration (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:315:5)       
        at Fixture.setup (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:58:19)
        at FixtureRunner.setupFixtureForRegistration (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:315:5)       
        at FixtureRunner.resolveParametersAndRunHookOrTest (...\playwrightdemo\node_modules\@playwright\test\lib\fixtures.js:300:23)
        at WorkerRunner._runTestWithBeforeHooks (...\playwrightdemo\node_modules\@playwright\test\lib\workerRunner.js:478:7)   
```

**Comment by elumalaik** [comment]
Would it be available in the next release? V1.17

**Comment by mxschmitt** [comment]
Version 1.18 includes fixes for Selenium 3 and 4. See [here](https://playwright.dev/docs/selenium-grid) for documentation and please file new bug reports if you still encounter bugs.
